test/models/transforms/test_input.py

```diff
@@ -1106,6 +1134,9 @@ def f2(x: Tensor, n_f: int = 1) -> Tensor:
             result = x[..., -2:].unsqueeze(-2)
             return result.expand(*result.shape[:-2], n_f, -1)
 
+        # set seed to range where this is known to not be flaky
+        torch.manual_seed(torch.randint(100, torch.Size([])).item())
```

Comment:
Nit: Can add `from random import randint` at the top of the file and then use
```suggestion
        torch.manual_seed(randint(0, 100))
```
which does not allocate an intermediate tensor just to extract an integer.